### ISSUE TITLE
Change the landing page text

### ISSUE THREAD

**Issue by sanchibansal340** [anitab-org/vms#1049]
## Description
As a user, I need the Systers to be replaced by Anita Borg from the landing page, volunteer and admin page.

## Acceptance Criteria
- [ ] Screenshot of the updated web pages.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour

## Note
This issue will be assigned from September 1, 2020.

**Comment by MayankVir** [comment]
I would like to work upon this issue for Open Source Hack!!

**Comment by khushishikhu** [comment]
can i work on this issue for OSH

**Comment by sanchibansal340** [comment]
Comment back on 1st September and I will assign you this issue.

**Comment by sanchibansal340** [comment]
Comment on 1st September and I will assign it to you.

**Comment by bhumikasharma29** [comment]
I would like to work on this.

**Comment by bhumikasharma29** [comment]
please assign it to me

**Comment by Abjcodes** [comment]
I would like to work on this issue as a part of OSH. Can you assign me this issue? Thank You.

**Comment by titiksha01** [comment]
Hello, I would like to work on this issue. Can this be assigned to me? Thank You

**Comment by Aaishpra** [comment]
Hey can u assign this to me

**Comment by sanchibansal340** [comment]
@bhumikasharma29 I am assigning it to you.

**Comment by bhumikasharma29** [comment]
Thank you @sanchibansal340

**Comment by alivabasa** [comment]
I want to work on this.

**Comment by HariniJeyaraman** [comment]
@sanchibansal340 Would love to work on this issue

**Comment by madhhuurrii** [comment]
I want to work on this issue as a first timer for OSH

**Comment by bhumikasharma29** [comment]
@sanchibansal340 I have cloned the project on my system. But since I am a new to this I was unable to find exactly where do I need to change the text. I cannot find Systers on the volunteer and admin page.

**Comment by pallavisavant** [comment]
i would like to work on this ,can this be assigned to me

**Comment by sanchibansal340** [comment]
> @sanchibansal340 I have cloned the project on my system. But since I am a new to this I was unable to find exactly where do I need to change the text. I cannot find Systers on the volunteer and admin page.

@bhumikasharma29 Were you able to find where the changes have to be made?

**Comment by bhumikasharma29** [comment]
@sanchibansal340 I did find the text to be changed on the landing page but someday else (muskanvk) added a commit for changing that although I was assigned the issue. And I could not find the places to make the change on admin or volunteer page.

**Comment by sanchibansal340** [comment]
@bhumikasharma29 The commit from the person who has been assigned the issue would be considered so don't worry about that. 
About the admin/volunteer page, you can ask about it on [Zulip](https://anitab-org.zulipchat.com/)

**Comment by bhumikasharma29** [comment]
Okay thanks!!

**Comment by ankit-saraf** [comment]
@sanchibansal340 Can I work on this issue?

**Comment by sanchibansal340** [comment]
@bhumikasharma29 Any updates?

**Comment by bhumikasharma29** [comment]
Yes I just want to confirm that the change has to be made under the Organization drop down list in the Sign up page?

**Comment by sanchibansal340** [comment]
I don't understand your question @bhumikasharma29 ?

**Comment by shubham582000** [comment]
I would like to work on this issue.
